doc/api_break.dox

```diff
@@ -1836,6 +1837,13 @@ QgsRuntimeProfiler        {#qgis_api_break_3_0_QgsRuntimeProfiler}
 
 - This class is no longer a singleton and instance() has been removed. Instead use QgsApplication::profiler() to access an application-wide profiler.
 
+
+QgsServer       {#qgis_api_break_3_0_QgsServer}
+----------
+
+-  QgsServer::handleRequest( const QString &urlstr ) now also takes two optional arguments: QgsServerRequest::Method and POST data
```

Comment:
Oh, I thought it's a QString -> enum change. In this case here, it's not really an API break, sorry ;)